src/dangerRender.ts

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable @typescript-eslint/no-shadow */
+// Provides dev-time typing structure for  `danger` - doesn't affect runtime.
+// https://github.com/danger/danger-js/blob/main/docs/usage/extending-danger.html.md#writing-your-plugin
+import { GenericTicketType, I_GenericTicket } from "./types";
+export declare function warn(message: string): void;
+export declare function markdown(message: string): void;
+export declare function schedule<T>(asyncFunction: Promise<T>): void;
+
+const StoryEmoji: Record<GenericTicketType, string> = {
+  chore: "⚙️",
+  epic: "⚡",
+  feat: "🌟",
+  fix: "🐞",
+};
+
+export const printWarningError = (e: Error): void => {
+  warn(e.message);
+};
+
+/**
+ * Comments with the ticket type, id and title
+ *
+ * @param ticket
```

Comment:
```suggestion
```

either constantly use complete jsDoc descriptions or don't use them at all (I favou the second option)